code/pizzabot.js

```diff
@@ -1,7 +1,266 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "vegetarian";
+const hawaiian = "hawaiian";
+const pepperoni = "pepperoni";
+const capricciosa = "capricciosa";
+const margherita = "margherita";
+const pizzaPrice = 80;
 
-const pizzaPrice = 80
+/* FUNCTIONS */
 
-//Put your Javscript code here:
+/* Calculate the cost of order */
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+  const orderTotal = orderQuantity * pizzaPrice;
+  return orderTotal;
```

Comment:
We can simplify this function. Since you are creating `const` variable and returning it very next line, we could declare it like this
```
const calculateTotalCost = (orderQuantity, pizzaPrice) => {
    return orderQuantity * pizzaPrice;
}
```
Another tip from my side - since we have only one line inside `calculateTotalCost` function, we can write it in "arrow function shorthand" :

`const calculateTotalCost = (orderQuantity, pizzaPrice) => orderQty * pizzaPrice;`

We can do that because in arrow functions we can omit curly brackets (`{}`) and `return` keyword if we only have one line of code inside. At the end, this snippet of code is exactly the same as the first one I wrote above.

For further reference on that subject, please check out https://www.w3schools.com/js/js_arrow_function.asp